### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by Imrhien** [nefarius/BthPS3#52] [Support]
Hi folks,

Might seem like an obvious question, but what do I do once the driver is installed? I've tried connecting my PS3 controller via the cable (doesn't work, although a USB connection noise occurs) and via Bluetooth (can't find the controller).

Is there some trick I'm missing?

**Comment by nefarius** [comment]
You're probably missing its companion [DsHidMini](https://vigem.org/projects/DsHidMini/How-to-Install/).

**Comment by Imrhien** [comment]
Works :) Thank you!
